### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by janicez11**
plz check the checkbox section.

### CODE REVIEW

File: journal/views/base.py [hunk 1]
```diff
@@ -24,12 +24,37 @@
...
+    if request.method == 'POST':
+        form = ContactForm(request.POST)
+
+        if form.is_valid():
+            subject = form.cleaned_data['subject']
+            message = form.cleaned_data['message']
+            sender = form.cleaned_data['sender']
+            cc_myself = form.cleaned_data['cc_myself']
+
+            recipients = ['info@example.com']
```
Comment (kevinlee12) [inline journal/views/base.py]: Can you do a comment at this line with the comment: `# TODO: Change recipients`

File: journal/views/base.py [hunk 2]
```diff
@@ -24,12 +24,37 @@
...
+                recipients.append(sender)
+
+            send_mail(subject, message, sender, recipients)
+            return HttpResponseRedirect('/thanks/')
+    else:
+        form = ContactForm()
+
     return render(request, 'journal/home.html',
-                  {'request': request, 'user': request.user})
+            {'request': request, 'user': request.user, 'form': form})
```
Comment (kevinlee12) [inline journal/views/base.py]: please start the `{` right below the request.

File: journal/views/base.py (hunk 1 above)
Comment (janicez11) [inline journal/views/base.py]: which line number ?

File: journal/views/base.py (hunk 2 above)
Comment (janicez11) [inline journal/views/base.py]: you mean like align the three 'request' ?

**Comment by kevinlee12** [comment]
@janicez11 , please merge from develop and push again. also, can you squash the two commits? that would be awesome. thanks!

File: journal/views/base.py (hunk 1 above)
Comment (kevinlee12) [inline journal/views/base.py]: Right after `['info@example.com'].

File: journal/views/base.py (hunk 2 above)
Comment (kevinlee12) [inline journal/views/base.py]: align with line 55 request.

**Pull request merged**